[READY FOR REVIEW] PPIE version of Inclusive Events chapter (#3550)
the-turing-way/the-turing-way: book/website/collaboration/ppie-events/ppie-events-resources.md

(cl-ppie-events-resources)=
# Resources

(cl-ppie-events-resources-learn)=
## What to Learn Next

> **Recommend other chapters in _The Turing Way_ book that readers should read next as a "level up", and explain why.**

> * {ref}`cm-lay-summaries` contains information and guidance on writing plain langugage summaries that canhelp guide the development of your plain langugage resources

Reviewer: ```suggestion
* {ref}`cm-lay-summaries` contains information and guidance on writing plain language summaries that can help guide the development of your plain language resources
```

> * {ref}`cl-organising-conferences` contains a checklist and general guidance for organizing a conferences which supplements this chapter if you are planning a longer or multi-day event

Reviewer: ```suggestion
* {ref}`cl-organising-conferences` contains a checklist and general guidance for organizing a conference which supplements this chapter if you are planning a longer or multi-day event
```

> * {ref}`cl-event-tools` contains suggestions and reccomendations for tools that can help with task management, collaborative note taking, live polls, and surveys which can be used to support your event

Reviewer: ```suggestion
* {ref}`cl-event-tools` contains suggestions and recommendations for tools that can help with task management, collaborative note taking, live polls, and surveys which can be used to support your event
```

> * {ref}`cl-remote-collab-guidelines` provides guidance on running remote meetings and has reccomendations that can help shape an online or remote event held for people with lived experience

Reviewer: ```suggestion
* {ref}`cl-remote-collab-guidelines` provides guidance on running remote meetings and has recommendations that can help shape an online or remote event held for people with lived experience
```
